rules/consistent-sharepoint-sites/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you have your SharePoint sites consistent?
+uri: consistent-sharepoint-sites
+authors:
+  - title: Warwick Leahy
+    url: warwick-leahy
+  - title: Jean Thirion
+    url: jean-thirion
+created: 2022-05-05T04:49:43.291Z
+guid: 55742b01-36da-491a-84d8-5855fee4f2bd
+---
+It's important that all of your SharePoint Sites are as consistent as possible.\
+\
+This helps the user to navigate a new page easily.  They need to know exactly where to look.
```

Comment:
Change from: This helps the user to navigate a new page easily. They need to know exactly where to look.
To: This helps users' navigation through new pages as they know exactly where to look